content/project/board-election-process.adoc

```diff
@@ -1,37 +1,135 @@
 ---
 layout: simplepage
-title: "Board election process"
+title: "Board and Officer election process"
 section: project
 ---
 
-This document outlines the link:/project/governance/#governance-board[Jenkins Governance Board] election process as discussed in governance meetings.
+This document outlines the link:/project/governance/#governance-board[Jenkins Governance Board] and link:/project/team-leads/[Jenkins officer] election process.
+The election process is driven by the Jenkins Governance board members who are not up for reelection (aka the _election committee_).
+More contributors might be added to the election committee if approved by the link:/project/governance-meeting/[regular governance meeting].
 
 The charter of the Jenkins board is to ensure the health of not only the software project, but of the communities of plugin developers and users. To do this effectively, members of the board must bring various perspectives to the table: what are the needs of users; of committers; of organizations, both large and small; of commercial interests. As the composition of the Jenkins board changes over time, we want to ensure that this balance of viewpoints is maintained. Current board members best understand this balance, and so are in the best position to select candidates for inclusion onto the board. A board comprised fully of write-in candidates runs the risk of being overweighed with one type of perspective over all others.
 
-The opinion of members of the community at large is highly valued, and the board welcomes additional nominations beyond who we might consider. If you feel that a particular person is well suited to help guide Jenkins, please submit a name and the reason for your nomination to jenkinsci-board@googlegroups.com. The current members of the board will review all submissions and compile a final list of candidates to be voted on by the community, for the seats up for election. The board reserves the right to leave nominations off the final ballot if they feel that a particular viewpoint is already well represented on the board.
-
 ## Goal
 
 . Make sure that the board reflects "the will of project participants" instead of just being self-appointed.
 . Make sure that the board reflects the balance of different viewpoints — the needs of users; of committers; of organizations, both large and small; of commercial interests.
 . Maintain stability and avoid sudden direction changes.
 
+## 2020 Elections schedule
+
+* **Sep 24** - Nominations open
+* **Sep 24** - Voting sign-up begins
+* **Oct 15** - Nominations deadline
+** After the deadline, the election committee will process nominations and reach out to candidates in order 
```

Comment:
I'm not clear on the intent of the phrase "in order".  Is that stating they will contact the candidates in the order that the candidates were nominated or is it a phrase that could be removed?

I assume this is intentionally notifying the selected candidates rather than notifying all nominees (which would include nominees that the board chose to not place as candidates)
```suggestion
** After the deadline, the election committee will process nominations and notify the candidates
```